src/Phinx/Console/CommandEvents.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Phinx\Console;
+
+class CommandEvents
```

Comment:
I can certainly see use cases for phinx.command.[migrate|rollback|seed|breakpoint].[started|completed|failed]

Well. Maybe not so much breakpoint, but keep the pattern as then you don't have "special" behaviour.

Also, (maybe overkill), would firing an event for each migration/seed be possible?

phinx.command.[migrate-single|rollback-single|seed-single].[started|completed|failed]

The event data for the singles (not a great name admittedly, so be inventive!!!) would need to include the migration name.

One use case for this for me would be that I could activate an "after" migration once a "before" migration has been executed.

Reply:
> I can certainly see use cases for phinx.command.[migrate|rollback|seed|breakpoint].[started|completed|failed]

I do that.

> Also, (maybe overkill), would firing an event for each migration/seed be possible?
> phinx.command.[migrate-single|rollback-single|seed-single].[started|completed|failed]
> The event data for the singles (not a great name admittedly, so be inventive!!!) would need to include the migration name.

If be honest I don't see any use case right now. I suggest not to develop that at present time.

> One use case for this for me would be that I could activate an "after" migration once a "before" migration has been executed.

You're right. Let's do it.